apps/gitea/src/main/fabric8/deployment.yml

```diff
@@ -5,7 +5,7 @@ spec:
   template:
     spec:
       containers:
-      - image: fabric8/gitea::${gitea.version}
+      - image: fabric8/gitea:${gitea.version}
```

Comment:
I know its in progress & not yet ready for prime time - any ideas @rajdavies or @rawlingsj